release/x86_64-freebsd10/Vagrantfile

```diff
@@ -81,6 +81,9 @@ Vagrant.configure(2) do |config|
   #   sudo apt-get install -y apache2
   # SHELL
   config.vm.provision "shell", inline: <<-SHELL
-    pkg install -y bash gmake ruby rubygem-bundler git-subversion python2
+    freebsd-update --not-running-from-cron fetch install
+    pkg upgrade -y
+    pkg install -y bash gmake ruby rubygem-bundler git-lite python2
+    gem update --system --no-document
```

Comment:
Is there a specific issue which you are hitting with the RubyGems that come packaged from FreeBSD?